yt_idv/shader_objects.py

```diff
@@ -252,6 +253,7 @@ class Shader(traitlets.HasTraits):
     )
     blend_equation = GLValue("func add")
     depth_test = GLValue("always")
+    _filename = traitlets.Unicode()
```

Comment:
added this attr to have a convenient way of printing error messages if compiling a shader from a file

Reply:
:+1: